analyzer/src/main/kotlin/managers/NPM.kt

```diff
@@ -263,7 +263,7 @@ open class NPM(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConf
                                     }
 
                                     hash = dist["shasum"].textValueOrEmpty()
-                                    hashAlgorithm = HashAlgorithm.SHA1
+                                    hashAlgorithm = HashAlgorithm.fromHash(hash)
```

Comment:
The Bundler implementation also uses a hardcoded algorithm, should the same change be done there, too?

Reply:
Thanks for catching, done.